src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -131,6 +131,13 @@ class ReportActionItemMessageEdit extends React.Component {
         Report.saveReportActionDraft(this.props.reportID, this.props.action.reportActionID, '');
         toggleReportActionComposeView(true, this.props.isSmallScreenWidth);
         ReportActionComposeFocusManager.focus();
+        // The listener below scrolls to the last comment once the keyboard is hidden to make sure the comment is not hidden and 
+        // does not go outside the viewable area in the scroll view. This only applies to the last/latest comment in the report.
+        if (this.props.index !== 0) { return; }
```

Comment:
```suggestion
        if (this.props.index === 0) {
```
and then put the `keyboardDidHideListener`... code inside the if block, then we wouldn't need the return

Reply:
Oh, maybe? I wasn't aware of that but I'm still new-ish to `App` - @Santhosh-Sellavel what do you think?

Reply:
Early return is fine!

Reply:
Cool - ignore this comment then!